src/hyp3_sdk/hyp3.py

```diff
@@ -592,25 +592,30 @@ def submit_aria_s1_gunw_job(
 
     @classmethod
     def prepare_aria_s1_gunw_job(
-        cls, reference: list[str], secondary: list[str], frame_id: int, name: str | None = None
+        cls, reference_date: str, secondary_date: str, frame_id: int, name: str | None = None
     ) -> dict:
         """Prepare an ARIA S1 GUNW job.
 
         Args:
-            reference: The names of the Sentinel-1 SLC granules to use as reference scenes for InSAR processing
-            secondary: The names of the Sentinel-1 SLC granules to use as secondary scenes for InSAR processing
+            reference_date: Date of reference scenes for InSAR processing in YYYY-MM-DD format
+            secondary_date: Date of secondary scenes for InSAR processing in YYYY-MM-DD format
             frame_id: Subset GUNW products to this frame
             name: A name for the job
 
         Returns:
             A dictionary containing the prepared ARIA S1 GUNW job
         """
         job_parameters = locals().copy()
-        for key in ['cls', 'reference', 'secondary', 'frame_id', 'name']:
+        for key in ['cls', 'reference_date', 'secondary_date', 'frame_id', 'name']:
```

Comment:
reference date, secondary date, and frame id can all pass as-is to the job parameters. We only need to pop fields that need special handling, e.g. translating `granule1=a` and `granule2=b` to granules: [a, b] for our other job types

```suggestion
        for key in ['cls', 'name']:
```